examples/systemcalls/systemcalls.c

```diff
@@ -9,13 +17,25 @@
 */
 bool do_system(const char *cmd)
 {
-
 /*
  * TODO  add your code here
  *  Call the system() function with the command set in the cmd
  *   and return a boolean true if the system() call completed with success
  *   or false() if it returned a failure
 */
+    int ret;
+
+    ret = system(cmd);
+
+    // check if system() executed correctly
+    if(-1 == ret)
+    {
+        return false;
+    }
+
+    // check for error in command execution
+    if(!WIFEXITED(ret))
+        return false;
```

Comment:
@ameyflash do these two checks ensure you will catch cases where cmd() returns error status values other than -1?